/var/spool/slurm/slurmd/job02422/slurm_script: line 20: 1619772 Killed                  ./merge_sort 10 1000000000 10000
Traceback (most recent call last):
  File "/users/vjoshi9/itcs-4145/merge-sort-/benchmark.py", line 18, in <module>
    ax = df.plot(
         ^^^^^^^^
  File "/users/vjoshi9/.conda/envs/myenv/lib/python3.12/site-packages/pandas/plotting/_core.py", line 995, in __call__
    elif not isinstance(data[x], ABCSeries):
                        ~~~~^^^
  File "/users/vjoshi9/.conda/envs/myenv/lib/python3.12/site-packages/pandas/core/frame.py", line 4107, in __getitem__
    indexer = self.columns.get_loc(key)
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/vjoshi9/.conda/envs/myenv/lib/python3.12/site-packages/pandas/core/indexes/range.py", line 417, in get_loc
    raise KeyError(key)
KeyError: 'size'
slurmstepd: error: Detected 1 oom_kill event in StepId=2422.batch. Some of the step tasks have been OOM Killed.
